build/build-docker-dev.sh

```diff
@@ -10,9 +10,15 @@ source "./build/verify-docker.sh"
 # freshly cloned whenever the SHA pointers are updated.
 ROCKSDB_SHA=$(git ls-tree HEAD _vendor/rocksdb | awk '{print $3}')
 ETCD_SHA=$(git ls-tree HEAD _vendor/src/github.com/coreos/etcd | awk '{print $3}')
-sed -e "s/ROCKSDB_SHA/${ROCKSDB_SHA}/g" \
+WANTED_CONTENT=$(sed -e "s/ROCKSDB_SHA/${ROCKSDB_SHA}/g" \
     -e "s/ETCD_SHA/${ETCD_SHA}/g" \
-    < build/devbase/Dockerfile.template > build/devbase/Dockerfile
+    < build/devbase/Dockerfile.template)
+if diff <(echo "${WANTED_CONTENT}") "build/devbase/Dockerfile" &> /dev/null; then
```

Comment:
Neat. I'm mildly surprised all of the whitespace is preserved precisely in `WANTED_CONTENT`. I usually use `cmp` instead of `diff` for testing equality.